CorsixTH/Lua/dialogs/staff_rise.lua

```diff
@@ -81,7 +81,7 @@ function UIStaffRise:UIStaffRise(ui, staff, rise_amount)
   self:makeTooltip(_S.tooltip.staff_window.salary, 14, 171, 168, 193)
   self:makeTooltip(_S.tooltip.staff_window.ability, 12, 213, 89, 243)
 
-  if profile.humanoid_class == "Doctor" then
+  if profile:isType("Doctor") then
```

Comment:
`profile` is derived from the `staff` parameter, so `class.is(staff, Doctor)` (or `self.staff` would work too).